inventory/group_vars/all.yml

```diff
@@ -51,12 +51,15 @@ cluster_name: cluster.local
 # but don't know about that address themselves.
 # access_ip: 1.1.1.1
 
-# Choose network plugin (calico, weave or flannel)
+# Choose network plugin (calico, weave, flannel or fan)
 kube_network_plugin: flannel
 
 # Kubernetes internal network for services, unused block of space.
 kube_service_addresses: 10.233.0.0/18
 
+# example underlay for ubuntu fan network
+#kube_network_fan_underlay: 10.11.0.0/16
```

Comment:
Sorry if i'm wrong but why don't you use the variable `kube_pods_subnet` ?
the kube_network_fan_underlay subnet is used to assign ip addresses to containers, right ?